### PR TITLE
Sync CHANGES.md and NEWS.md with 3.0.8 release

### PR DESCRIPTION
**Pull request by t8m**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -19699,6 +19861,17 @@ ndif
...
+[CVE-2023-0286]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0286
+[CVE-2023-0217]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0217
+[CVE-2023-0216]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0216
+[CVE-2023-0215]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0215
+[CVE-2022-4450]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4450
+[CVE-2022-4304]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4304
+[CVE-2022-4203]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4203
+[CVE-2022-3996]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-3996
+[CVE-2022-2274]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-2274
+[CVE-2022-2097]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-2274
```
Comment (mattcaswell) [inline CHANGES.md:19874]: Copy & paste error here (2097 vs 2274)

File: NEWS.md [hunk 2]
```diff
@@ -1443,7 +1455,17 @@ OpenSSL 0.9.x
...
+[CVE-2023-0286]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0286
+[CVE-2023-0217]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0217
+[CVE-2023-0216]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0216
+[CVE-2023-0215]: https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0215
+[CVE-2022-4450]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4450
+[CVE-2022-4304]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4304
+[CVE-2022-4203]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-4203
+[CVE-2022-3996]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-3996
+[CVE-2022-2274]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-2274
+[CVE-2022-2097]: https://www.openssl.org/news/vulnerabilities.html#CVE-2022-2274
```
Comment (mattcaswell) [inline NEWS.md:1468]: Same error here

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (t8m) [reply CHANGES.md:19874]: Hmm, this is in 3.0 branch as well. I'll use a separate commit for that so it can be cherry-picked.

**Comment by t8m** [comment]
The second commit needs to be cherry-picked to 3.0 branch.

**Review by paulidale — APPROVED** [review]

**Comment by mattcaswell** [comment]
CI failure is unexpected - but unrelated to this PR.

**Review by mattcaswell — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master and 3.1 and the second commit to 3.0 as well. Thank you for the reviews.

**Pull request closed without merging**